scripts/scil_bundle_fixel_analysis.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Analyze bundles at the fixel level using peaks and bundles (.trk). If the
+bundles files are names as {bundle_name}.trk, simply use the --in_bundles
+argument with --in_bundles_names. If it is not the case or you want other names
+to be saved, please use --in_bundles_names to provide bundles names IN THE SAME
+ORDER as the inputed bundles.
```

Comment:
Can you give the user un feeling for the time this script will take? I just ran this line
`scil_bundle_fixel_analysis.py 102311/FODF_Metrics/102311__peaks.nii.gz --in_bundles results_rbx/102311/Clean_Bundles/102311__*trk --in_bundles_names AF_L AF_R CC_Fr_1 CC_Fr_2 CC_Oc CC_Pa CC_Pr_Po CC_Te CG_L CG_R FAT_L FAT_R FPT_L FPT_R FX_L FX_R IFOF_R ILF_L ILF_R MCP MdLF_L MdLF_R OR_L OR_R POPT_L POPT_R PYT_L PYT_R SCP_L SCP_R SLF_L SLF_R --processes 6 -v`

I don't have any output and any feeling if my input line is ok or how long it will take?

Any chance you could quickly check the in_bundles and in_bundles_names coherence? Are you checking if they have the same size for instance?

Reply:
It really depends on the number of bundles and the number of streamlines in them. I will add a check for the bundles names. Also, I will add a bit of verbose for each big steps.